<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Anytest</class>
 <widget class="QWidget" name="Anytest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amplitude</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Phrase</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>61</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>VA:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>VB:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>VC:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>VZ:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>IA:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>IB:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>IC:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_12">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_13">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_16">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_17">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_18">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>240</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_19">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_20">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_21">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>270</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>221</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Step setting</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>current(A)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>voltage(V)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>phrase(Â°)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>frequency(Hz):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>step time(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>step number</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_27">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_23">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_22">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_24">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_26">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_25">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>360</y>
     <width>271</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>fault time setting</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>fault(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>pre(s)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>post(s):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Fault duration</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_30">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_29">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_28">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_41">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>211</width>
     <height>311</height>
    </rect>
   </property>
   <property name="title">
    <string>variable</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>VA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>VB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>VC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>VZ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>VZ=3U0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>IA</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>IB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>IC</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_32">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_31">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_33">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3U0">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_37">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_35">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_36">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_27">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>test mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_28">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>400</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>change</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_38">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_39">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>400</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>550</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>22</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_40">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
